developerportal/templates/organisms/featured.html

```diff
@@ -7,13 +7,13 @@ <h2>Featured</h2>
   {% if featured_count != 2 %}
     {# first item is rendered as a primary card #}
     {% with featured.0 as block %}
-      {% if block.block_type == 'post' or block.block_type == 'event' %}
+      {% if block.block_type == 'post' or block.block_type == 'event' or block.block_type == 'video' %}
```

Comment:
nit: For comparing with more than 2 I like to write (and read):
```python
if block.block_type in ('post', 'event', 'video')
```

Reply:
This is another good reason to switch to Jinja2, I think 👍